### DISCUSSION TITLE
Ipad compatibility issue after update

### DISCUSSION THREAD

**Discussion by daanha** [Lurkars/gloomhavensecretariat#551] [Q&A]
Hi, I use an older ipad pro for Frosthaven, and someone from our party hosts it on his NAS. After the latest update, the monster ability cards are no longer visible. They just flip upside-down. Same for the loot cards. Is there any chance that this could be fixed? 

I use Chrome on the ipad, the native Safari browser won't even open the site anymore...

Thanks, Daan

**Comment by Lurkars** [comment]
Can you please provide a version of GHS (in the about menu) and are you able to check other devices? If newer iOS devices has no issues, I am unsure how I will be able to fix this. (Btw: not related to this, right? https://github.com/Lurkars/gloomhavensecretariat/issues/533)

**Comment by daanha** [comment]
Hi, we use GHS 0.95.5.

It certainly seems related to that issue. I thought it could be something
with the animation of the card flipping, but disabling that didn't help...

Thanks for your reply, I can imagine that debugging the iOs versions is
difficult.

Op di 9 apr. 2024 07:57 schreef Lurkars ***@***.***>:

**Reply by Lurkars** [reply]
So I can only test myself on an iPad 7th Gen, current iPadOS 16.31, Chroms version is 123.0 and I cannot reproduce the issue. It's working fine for me. Since the cards are styled with basic CSS technic for the flip-effect (not only the animation), I am afraid that I can only tell you that too old devices are just not supported...

**Comment by daanha** [comment]
Okay, thanks for checking!

Daan

Op di 9 apr. 2024 17:11 schreef Lurkars ***@***.***>:

**Comment by daanha** [comment]
I have one more question: on my phone (Android samsung s10) if touch the
loot pile it takes me to the loot overview screen (see attachment). I can't
draw a loot card, no matter where I press on the card.

Do you know about this issue?

Thanks, Daan

Op di 9 apr. 2024 17:22 schreef Daan Haeyen ***@***.***>:

**Reply by Lurkars** [reply]
Are you in a round or is the "Draw" button visible? Because deck is disabled when not in a round.

**Comment by daanha** [comment]
Yes, it's also when I'm in a round.

I solved the ipad compatibility issue by the way, it turned out I could
update it to a newer ios!

Thanks, Daan

Op do 11 apr. 2024 18:32 schreef Lurkars ***@***.***>:

**Reply by Lurkars** [reply]
Okay, I didn't get the screenshot by the way. So maybe post again here or write an actual email to support@gloomaven-secretarait.de, so I can take a look.
